lms/djangoapps/bulk_email/tasks.py

```diff
@@ -159,7 +159,7 @@ def _get_course_email_context(course):
         'course_url': course_url,
         'course_image_url': image_url,
         'course_end_date': course_end_date,
-        'account_settings_url': 'https://{}{}'.format(settings.SITE_NAME, reverse('account_settings')),
+        'email_settings_url': 'https://{}{}'.format(settings.SITE_NAME, reverse('dashboard')),
```

Comment:
No, I didn't know about edx-emails. How does the stuff in that repo relate to the templates in this PR?